typescript/cli/src/utils/keys.ts

```diff
@@ -1,5 +1,6 @@
 import { input } from '@inquirer/prompts';
 import { ethers, providers } from 'ethers';
+import { Wallet } from 'zksync-ethers';
```

Comment:
ditto as above, confused by why we need to use the zk-specific wallet for the entire CLI here

Reply:
Agree, its confusing and far cry from ideal solution. In short, its backwards compatible with ethers wallet. We had discussions about limitations of the current Signer related features and its compatibility with MultiProvider, which is especially important for the upcoming Starknet SDK support.

Currently in `getContext` the value of `signer` is passed to `getMultiProvider` which in turn sets shared signer across all chains in the registry. At the time of getting singer, we don't know the protocol or chain technical stack that we can use to set the correct type of Signer.

Reply:
@jmrossy We created a PR to address this issue https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4914/files